.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+name: Lint code & Run tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@stable
+  
+  build:
+    needs: lint
+    runs-on: Ubuntu-20.04
+
+    env:
+      DB_TYPE: postgresql
+      DB_USERNAME: postgres
+      DB_PASSWORD: postgres
+      DB_ENDPOINT: localhost:5432 
+      DB_TEST_NAME: stem_diverse_test
+      
+    services:
+      postgres:
+        image: postgres
+        env: 
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Connect to PostgreSQL
+      run: |
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE DATABASE stem_diverse_test;' -U postgres
```

Comment:
You can pass database name in postgres image above when creating postgres container. No need to run this here

Reply:
Yeah but this would stlil work. If you are requesting this change then you might as well run the tests inside a docker container.

Reply:
Running tests in docker container is not required. However, this is the way initial database is defined in a postgres service container

Reply:
> Running tests in docker container is not required. However, this is the way initial database is defined in a postgres service container

It won't be any different. I'm guessing docker would do the exact same thing under the hood.

Reply:
Should I make any changes? 👀 @devkapilbansal @epicadk